server/controllers/stock/core.js

```diff
@@ -389,27 +389,62 @@ function getLotsOrigins(depotUuid, params) {
  *
  * @description
  * Stock Management Processing
+ *
+ * DEFINITIONS:
+ *   S_SEC: Security Stock - one month of stock on hand based on the average consumption.
+ *   S_MIN: Minimum stock - twice the security stock.
+ *
+ *   S_RP: Risk of Expiration.
  */
 function stockManagementProcess(inventories) {
-  const current = moment();
   let CM;
   let Q;
   let CM_NOT_ZERO;
-  let delay;
 
   for (let i = 0; i < inventories.length; i++) {
     const inventory = inventories[i];
+
+    // the quantity of stock available in the given depot
     Q = inventory.quantity; // the quantity
+
+    // Average Monthly Consumption (CMM/AMC)
+    // This is calculuated during the stock exit to a patient or a service
+    // It is _not_ the average of stock exits, as both movements to other depots
+    // and stock loss are not included in this.
     CM = inventory.avg_consumption; // consommation mensuelle
     CM_NOT_ZERO = !CM ? 1 : CM;
+
+    // Compute the Security Stock
+    // Security stock is defined by taking the average monthly consumption (AMC or CMM)
+    // and multiplying it by the Lead Time (inventory.delay).  The Lead Time is by default 1 month.
+    // This gives you a security stock quantity.
     inventory.S_SEC = CM * inventory.delay; // stock de securite
+
+    // Compute Minimum Stock
+    // The minumum of stock required is double the security stock.
     inventory.S_MIN = inventory.S_SEC * 2; // stock minimum
+
```

Comment:
We should probably make that '2' a parameter.  And should it be possible to assign different values to different inventory items?